components/list/nz-list.component.ts

```diff
@@ -130,16 +103,16 @@ export class NzListComponent implements OnChanges {
       [ `${this.prefixCls}-something-after-last-item` ]: !!(this.nzLoadMore || this.nzPagination || this._isFooter)
     };
     this.updateHostClassService.updateHostClass(this.el.nativeElement, classMap);
-
-    this.cd.detectChanges();
   }
 
-  // endregion
+  // #endregion
 
   constructor(private el: ElementRef, private cd: ChangeDetectorRef, private updateHostClassService: NzUpdateHostClassService) {
   }
 
-  ngOnChanges(changes: SimpleChanges): void {
+  ngOnChanges(): void {
     this._setClassMap();
+    // Delayed detection refresh limited by nz-spin (issesus (#11280)[https://github.com/angular/material2/issues/11280) )
```

Comment:
"issesus" misspelled, fixed it to allow merge

Reply:
And "[https://github.com/angular/material2/issues/11280)" tobe "[https://github.com/angular/material2/issues/11280]"